content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -91,7 +91,7 @@ To see which admission plugins are enabled:
 kube-apiserver -h | grep enable-admission-plugins
 ```
 
-In 1.18, they are:
+In 1.19, they are:
```

Comment:
We often  handle this like:
```suggestion
In Kubernetes {{< skew latestVersion >}}, they are:
```
and put an onus on the relevant SIG(s) to ensure that the list is accurate.

However, I like https://github.com/kubernetes/website/pull/24878/files#r518137284 better.

Reply:
This is the list of plugins enabled _by default_, right?

Reply:
@sftim Yes, this list comes from here 🙂  :  https://github.com/kubernetes/kubernetes/blob/e7b9453972eccaa57d18dfb7024b017b97de0b26/pkg/kubeapiserver/options/plugins.go#L144